services/welding-machine-data-simulator-service/app/services/azure_storage.py

```
@@ -0,0 +1,178 @@
+import pandas as pd
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)